.github/workflows/_test-docker.yaml

```diff
@@ -38,3 +38,17 @@ jobs:
       image_name: test-docker-build
       docker_context: tests/docker
       artifact_retention_days: 1
+
+  test_docker_build_secrets:
+    uses: ./.github/workflows/docker-build-push-ecr.yaml
+    secrets:
+      docker_secrets: |
+        SECRET_TOKEN="S€cReT1!"
```

Comment:
I would remove this since people might actually do it like this :D Add a secret to to the repo and reference it.

Reply:
Good idea :smile: